### DISCUSSION TITLE
How to manage IPV6 devices in Adguardhome?

### DISCUSSION THREAD

**Discussion by mphin** [AdguardTeam/AdGuardHome#5821] [Q&A]
Network device situation:
Adguardhome is installed on the side router, and the main router is responsible for DHCP. Currently, IPV4 can manage devices statically by setting DHCP through the main router.
Attempt:
1. IPV6 sets DHCP static, but the prefix of the ISP can change at any time, resulting in changes in the client address.
2. Manage through stateless IPV6 addresses, similar to fe80:, This is useful, but after a few days of use, the stateless IPV6 address will change only after the IOS device restarts, making it impossible to manage IPV6 devices on the adguardhome.
So how do I manage IPV6 addresses on Adguardhome?

**Comment by lucasmz-dev** [comment]
You can use ULAs and NAT, but I personally don't recommend it as potentially your IPv6 connection won't be preferred. (AFAIK)
Also NAT on IPv6 is shamed upon. But if it's the only way to go, then hey.

I'm unsure if that's possible with Adguard Home however.

You should get ask your ISP to give you a static prefix. It's wrong for it to change, especially while your CPE is online, as that will announce new addresses to your devices but keep the old ones and breaking connectivity.

**Reply by mphin** [reply]
Thank you, there are indeed too many DNS issues with IPV6 management. I have set the DNS settings of IPV6 to non-existent addresses to force clients to use IPV4. However
